packages/devtools/web/styles.css

```diff
@@ -530,6 +530,7 @@ div.tabnav {
     padding: 0 2px;
     position: absolute;
     overflow-x: hidden;
+    white-space: pre;
```

Comment:
what about the frame-timeline benefits from preserving whitespace? Generally it is nice to write html ignoring whitespace using padding and margin instead of explicit whitespace.

Reply:
If we don't have this, the event names will wrap and it will cause the div to be taller than one row, which is not what we want.

Reply:
`white-space: nowrap;`
is more idiomatic if all you want to do is avoid wrapping.

Reply:
done.